test/Ocelot.UnitTests/Consul/ConsulFileConfigurationRepositoryTests.cs

```diff
@@ -1,19 +1,19 @@
-﻿using global::Consul;
-using Microsoft.Extensions.Options;
-using Moq;
-using Newtonsoft.Json;
-using Ocelot.Cache;
-using Ocelot.Configuration.File;
-using Ocelot.Logging;
-using Ocelot.Provider.Consul;
-using Ocelot.Responses;
-using Shouldly;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading;
-using System.Threading.Tasks;
-using TestStack.BDDfy;
+﻿using global::Consul;
+using Microsoft.Extensions.Options;
+using Moq;
+using Newtonsoft.Json;
+using Ocelot.Cache;
+using Ocelot.Configuration.File;
+using Ocelot.Logging;
+using Ocelot.Provider.Consul;
+using Ocelot.Responses;
+using Shouldly;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using TestStack.BDDfy;
 using Xunit;

```

Comment:
I don't see any changes!
Why did you include this file to the PR?
Have you changed encoding of the file aka code page?

Reply:
Line breaks I presume... Should have checked...

Reply:
Do you code in Visual Studio or Visual Code?
And what's OS, Linux or Windows?

Reply:
Visual Studio 22 on Windows